### PR TITLE
Clean up links

### PR DESCRIPTION
**Pull request by phlptp**
### Summary

If merged this pull request will clean up some links on the readme and user guide pages

### CODE REVIEW

**Comment by phlptp** [comment]
there is likely more to come but I needed the docs to build first

File: README.md [hunk 1]
```diff
@@ -80,32 +81,12 @@ Additionally, our initial design requirements document can be found [here](docs/
...
-HELICS can be installed on Linux (and macOS) using Spack, a package manager aimed at HPC environments. See [spack install](https://helics.readthedocs.io/en/latest/installation/package_manager.html#install-using-spack-macos-linux) for the instructions.
+A snapshot of some current plans for what is to come.
 
-### Building from Source
-
-- [Windows](https://docs.helics.org/en/latest/user-guide/installation/windows.html)
-- [Mac](https://docs.helics.org/en/latest/user-guide/installation/mac.html)
-- [Ubuntu](https://docs.helics.org/en/latest/user-guide/installation/linux.html)
 
 ### Docker
```
Comment (nightlark) [inline README.md:91]: This section looks like it should be removed (moved to the installation guide on RTD), moved up a level to `## Docker`, or have the `## Installation` header left in (maybe with a quick word and a link to the installation guide to make install instructions more prominent).

The table of contents at the top of the README also needs updating to reflect the changes.

File: docs/user-guide/advanced_topics/advanced_topics_index.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
     orchestration_monte_carlo
     program_termination
     profiling
+    targetted_endpoints
```
Comment (nightlark) [inline docs/user-guide/advanced_topics/advanced_topics_index.md:21]:
```suggestion
    targeted_endpoints
```

File: docs/user-guide/fundamental_topics/fundamental_topics_index.md [hunk 3]
```diff
@@ -23,11 +23,18 @@ The topics considered "fundamental" to building a co-simualtion with HELICS are:
...
+  - [**With JSON config file**](./interface_configuration.md#json-configuration)
+  - [**With HELICS APIs**](./interface_configuration.md#api-configuration)
 - [**Timing Configuration**](./timing_configuration.md) - How HELICS coordinates the simulation time of all the federates in the federation
+  - [Timing Exercise](./timing_exercise.md)
+  - [Timing Exercise answers](./timing_exercise_answers.md)
 - [**Stages of the Co-simulation**](./stages.md)
 - [**Logging**](./logging.md) - Discussion of logging within HELICS and how to control it.
 - [**Execution with `helics_cli`**](./helics_cli.md) - The HELICS team has developed `helics_cli` as a standardized means of running HELICS co-simulations.
 - [**Simulation Management**](./web_interface.md) - Using the webserver, HELICS also has a built-in web interface for running, monitoring, and diagnosing co-simulations.
+- [**Simulator Integration**](./simulator_integration.md) - A guide for integrated HELICS into simulators.
```
Comment (nightlark) [inline docs/user-guide/fundamental_topics/fundamental_topics_index.md:40]:
```suggestion
- [**Simulator Integration**](./simulator_integration.md) - A guide for integrating HELICS into simulators.
```

File: docs/index.md [hunk 4]
```diff
@@ -17,6 +17,12 @@ Brief History: HELICS began as the core software development of the Grid Moderni
 
 Motivation: Energy systems and their associated information and communication technology systems are becoming increasingly intertwined. As a result, effectively designing, analyzing, and implementing modern energy systems increasingly relies on advanced modeling that simultaneously captures both the cyber and physical domains in combined simulations. It is designed to increase scalability and portability in modeling advanced features of highly integrated power system and cyber-physical energy systems.
 
+## Quick links
+- [configuration option reference](./references/configuration_options_reference.md)
```
Comment (nightlark) [inline docs/index.md:21]:
```suggestion
- [Configuration Options Reference](./references/configuration_options_reference.md)
```

**Review by nightlark — APPROVED** [review]

**Pull request merged**